spec/fixtures/users.yml

```diff
@@ -134,3 +143,5 @@ affiliate_requiring_manual_approval:
   <<: *DEFAULTS
   email: affiliate_requiring_manual_approval@email.gov
   requires_manual_approval: true
+  approval_status: not_approved
```

Comment:
Ah yes this was the fixture I found that seemed to be incorrect because it had the approval status as approved even though it required manual approval. I was going to fix it while I was in the code. Did not know about pending_approval as one of the status's will correct.